netlify-ignore.sh

```diff
@@ -28,6 +28,12 @@ case $TARGET_PACKAGE in
         status=$?
         ;;
 
+    xstate-wallet)
```

Comment:
We probably don't need this netlify-ignore script since packages fall into 2 categories:
- Manual deploys via `deploy` branch.
- CI deploys where a package is built on Circle and release is triggered on Circle.